src/scripts/services/StatusService.js

```diff
@@ -1,15 +1,27 @@
+import logger from '@components/Logger';
+
 import HttpService from '@services/HttpService';
 import User from '@models/User';
 
 import { API_URL } from '@config';
 
+const handleErrors = async (response) => {
```

Comment:
Suggest renaming to something like handleOutput or processData as it also processes the case when everything is ok.

Reply:
Also method handleErrors  appears in other services, i think it would be better to move this method into one place and reuse it.